crates/miden-objects/src/transaction/inputs/mod.rs

```diff
@@ -25,7 +25,7 @@ pub use notes::{InputNote, InputNotes, ToInputNoteCommitments};
 /// Contains the data required to execute a transaction.
 #[derive(Debug, Clone, PartialEq, Eq)]
 pub struct TransactionInputs {
-    account: Account,
+    partial_account: PartialAccount,
```

Comment:
nit: I'm not sure there is a need to rename this to `partial_account`. The type already conveys this information and just `account` is more concise. This is also a bit more consistent with other naming here (e.g., we have `block_chain: PartialBlockchain`).